### PR TITLE
Marked the Mono.WebAssembly.Interop package as non-shipping

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

**Review by pranavkm — APPROVED** [review]

File: src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj
Comment (dougbu) [inline src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj:9]: 1. Changing `$(IsPackable)` will break the Microsoft.AspNetCore.Blazor.Templates.csproj template. Suggest that template should either be deleted or removed from the project templates meta-package (if it hasn't already).
2. What are we keeping this project for? I don't see references to it other than in Microsoft.AspNetCore.Blazor.Templates.csproj and general infrastructure files e.g. eng/ProjectReferences.props. Suggest cleaning up that file and ComponentsNoDeps.slnf.
3. When `$(IsPackable)` is `false`, changing `$(IsShipping)` mainly controls versions. Does the version matter?

Comment (mkArtakMSFT) [reply src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj:9]: The only thing which matters at this point is to make this not ship any more. Are you saying when `IsPackable` is `false` the package will be shipped anyway?

I don't want to do more than this for now as things will be resolved after blazor-wasm will merge into `master` (eventually)

Comment (dougbu) [reply src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj:9]: > Are you saying when `IsPackable` is `false` the package will be shipped anyway?

No. But, the project has a version which will show up in the Blazor template and break anyone trying to use it. In addition, the version will remain the same when shipping versions stabilize.

File: src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj
Comment (mkArtakMSFT) [inline src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj:8]:
```suggestion
```

File: src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj
Comment (mkArtakMSFT) [reply src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj:9]: Ok. That template is also currently marked to not ship. So I think simply marking this as `non-shipping` and leaving the `IsPackable` as is resolves your concern.

**Review by dougbu — APPROVED** [review]

**Pull request merged**
